### PR TITLE
 Update changelog for 1.2.0 release

### PR DESCRIPTION
**Pull request by FlorianVeaux**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 Changes
 =======
+## 1.2.0 / 2020-07-23
```
Comment (sarah-witt) [inline CHANGELOG.md:3]: The formatting is a little different than the other entries. I'm not sure if it's best to start using the automated format or our old one

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+
+* [Added] Use inclusive naming . See [#94](https://github.com/jenkinsci/datadog-plugin/pull/94).
```
Comment (sarah-witt) [inline CHANGELOG.md:5]: Nit- some of the spacing is off 
```suggestion
* [Added] Use inclusive naming. See [#94](https://github.com/jenkinsci/datadog-plugin/pull/94).
```

File: CHANGELOG.md (hunk 1 above)
Comment (FlorianVeaux) [reply CHANGELOG.md:3]: Let's keep the current one for now, good catch

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
 Changes
 =======
+## 1.2.0 / 2020-07-23
+
+* [Added] Use inclusive naming. See [#94](https://github.com/jenkinsci/datadog-plugin/pull/94).
+* [Added] Add new queue metrics with job tags. See [#73](https://github.com/jenkinsci/datadog-plugin/pull/73).
+* [Added] Add new node status metrics . See [#71](https://github.com/jenkinsci/datadog-plugin/pull/71).
```
Comment (sarah-witt) [inline CHANGELOG.md:7]:
```suggestion
* [Added] Add new node status metrics. See [#71](https://github.com/jenkinsci/datadog-plugin/pull/71).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+* [Fixed] Keep backwards compatibility with old config naming. See [#95](https://github.com/jenkinsci/datadog-plugin/pull/95).
+* [Fixed] Don't fail validation if log collection is not working. See [#90](https://github.com/jenkinsci/datadog-plugin/pull/90).
+* [Fixed] Re-order configuration loading logic. See [#87](https://github.com/jenkinsci/datadog-plugin/pull/87).
+* [Fixed] Fix buildable and pending metrics. See [#83](https://github.com/jenkinsci/datadog-plugin/pull/83).
+* [Fixed] Update failedlastValidation if client passes validation  . See [#81](https://github.com/jenkinsci/datadog-plugin/pull/81).
```
Comment (sarah-witt) [inline CHANGELOG.md:12]:
```suggestion
* [Fixed] Update failedlastValidation if client passes validation. See [#81](https://github.com/jenkinsci/datadog-plugin/pull/81).
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,19 @@
 Changes
 =======
+## 1.2.0 / 2020-07-23
+
+* [Added] Use inclusive naming. See [#94](https://github.com/jenkinsci/datadog-plugin/pull/94).
```
Comment (FlorianVeaux) [inline CHANGELOG.md:5]:
```suggestion
# 1.2.0 / 2020-07-23
### Details
https://github.com/jenkinsci/datadog-plugin/compare/datadog-1.1.2...datadog-1.2.0

### Changes
* [Added] Use inclusive naming. See [#94](https://github.com/jenkinsci/datadog-plugin/pull/94).
```

File: CHANGELOG.md (hunk 1 above)
Comment (FlorianVeaux) [reply CHANGELOG.md]: Let's keep the current one for now, good catch

**Pull request merged**